lib/reverse_sentence.rb

```diff
@@ -1,6 +1,33 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n ** c), the time complexity is cuadratic because there is a nested loop which iterations depend on the size of the string
+# Space complexity: O(n), where n is the size of the array that will be created to store the pieces of the new string
 def reverse_sentence(my_sentence)
```

Comment:
Your tests appear to not be passing purely because you aren't copying `reverse_sentence` back into `my_sentence`.